README.md

```diff
@@ -1,3 +1,22 @@
+# This fork is working to incorporate encoders, oled displays, and haptic feedback
```

Comment:
This blurb makes context in your own repo, but doesn't make sense to be upstreamed. However, this information is still valuable to maintain - maybe it should be moved somewhere in `/doc`? Open to ideas